doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -519,7 +519,7 @@ but moderate. The minimum timeouts should never be below 10 seconds.
 Location to store the resource state in.
 </longdesc>
 <shortdesc lang="en">State file</shortdesc>
-<content type="string" default="/var/run//Dummy-{OCF_RESOURCE_INSTANCE}.state" />
+<content type="string" default="/var/run//Dummy-default.state" />
```

Comment:
No, this is incorrect.

We removed the '$' on purpose since the metadata call cannot see the future and know what someone will call a resource that doesn't exist yet.  It was expected that people would be able to approximate the shell expansion in their head.

Possibly it should be s/OCF_RESOURCE_INSTANCE/name of your resource/ but 'default' is both wrong and misleading.